package.json

```diff
@@ -41,9 +41,11 @@
     "concurrently": "^7.4.0",
     "constructs": "^10.0.0",
     "date-fns": "^2.29.3",
+    "lodash": "^4.17.21",
     "pretty-bytes": "^6.0.0"
   },
   "devDependencies": {
+    "@types/lodash": "^4.14.191",
```

Comment:
Lodash was added twice. Presumably not needed in devDependecies?

Reply:
Without "@types/lodash", Typescript can't find a declaration file for module 'lodash'. Otherway would be manually declare module lodash in types.d.ts but this faster

Reply:
Reading comprehension fail

Reply:
Or I misunderstood your question

Reply:
The fail was on me. Didn't notice the @types